docs/chart-and-series-types/sunburst-series.md

```diff
@@ -1,7 +1,7 @@
 Sunburst
 ===
 
-A Sunburst displays hierarchical data, where a level in the hierarchy is represented by a circle. The center represents the root node of the tree. The visualization bear a resemblance to both treemap and pie charts.
+A Sunburst displays hierarchical data, where a level in the hierarchy is represented by a circle. The center represents the root node of the [tree](/docs/chart-concepts/dataviz-glossary#tree-data-structure). The visualization bear a resemblance to both treemap and pie charts.
```

Comment:
We've leaned towards using the full URL for internal links before, so they work when viewed from GitHub (or just raw markdown)

Reply:
Oh, I didn't know that. It was mixed in the existing docs and a liked the shorter version but I will change all links. Is this info added somewhere or maybe you know a place where it would fit?

Reply:
[Enforced](https://github.com/highcharts/highcharts/pull/22582/files#diff-6da33b3151b6f0cfd32bdbf35f80aae5dd24f2ad5460358111099646e722f31c)

Reply:
We don't seem to have this written down anywhere, adding a task on it on Asana